integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiFmwSample.java

```diff
@@ -47,21 +47,19 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
There is significant complex duplicate code between ItMIIFmwSample and ItMIIWlsSample, including variable declaration, initialization, and helper methods.

Reply:
Can the common information in the two classes be moved to a shared parent class?

Reply:
I removed the two classes ITMiiFmwSample and ItMiiWlsSample and restored the ItMiiSample.  I can skip WLS tests in FMW image and vice versa using environment variable  SKIP_WLS(JRF)_SAMPLES